File: FizzBuzz/FizzBuzz.py
def FizzBuzzFirst(max_loop: int) -> str:
    fizz_buzz = ""
    for i in range(1, max_loop+1):
        if i%3 == 0 and i%5 == 0:
            fizz_buzz += " FizzBuzz "
        elif i%3 == 0:
            fizz_buzz += " Fizz "
        elif i%5 == 0:
            fizz_buzz += " Buzz "
        else:
            fizz_buzz += f" {str(i)} "
    return fizz_buzz.strip()

def FizzBuzzOneLine(max_loop: int) -> str:
    return "".join([" FizzBuzz " if i%3 == 0 and i%5 == 0 else (" Fizz " if i%3 == 0 else (" Buzz " if i%5 == 0 else (f" {str(i)} "))) for i in range(1, max_loop+1)]).strip()

if __name__ == '__main__':
    print(FizzBuzzFirst(100))
    print(FizzBuzzOneLine(100))

File: FizzBuzz/FizzBuzz.go
package main

import (
	"fmt"
	"strings"
)

func FizzBuzz(maxLoop int) string {
	var fizzBuzz string
	for i := 1; i <= maxLoop; i++ {
		if i%3 == 0 && i%5 == 0 {
			fizzBuzz += " FizzBuzz "
		} else if i%3 == 0 {
			fizzBuzz += " Fizz "
		} else if i%5 == 0 {
			fizzBuzz += " Buzz "
		} else {
			fizzBuzz += fmt.Sprintf(" %d ", i)
		}
	}
	return strings.Trim(fizzBuzz, " ")
}

func main() {
	fmt.Println(FizzBuzz(100))
}
